kotlin/src/test/kotlin/org/jdbi/v3/core/kotlin/KotlinMapperTest.kt

```diff
@@ -40,6 +40,22 @@ class KotlinMapperTest {
 
     }
 
+    @Test fun shouldUseSecondOccurrenceOfColumnWhenMultipleColumnsWithSameNameArePresent() {
+
+        val mapper = KotlinMapper(DataClassWithOnlyPrimaryConstructor::class.java)
+        mockColumns("id", "name", "name")
```

Comment:
I thought that it would be a good idea to have tests showing 

- Mapper can handle duplicate columns
- The behaviour is deterministic, i.e. columns are not chosen at random
- Explicitly show which column will be used

I think the standard BeanMapper actually behaves in the same way.

If you think that we do not want to make any promises regarding mapping of duplicate columns, I will be happy to remove these tests.